sdk/tables/ci.yml

```
@@ -10,6 +10,6 @@ pr:
     - sdk/tables/
 
 stages:
-- template: ../../../eng/pipelines/templates/jobs/archetype-sdk-client.yml
+- template: ../../eng/pipelines/templates/jobs/archetype-sdk-client.yml
```

```suggestion
- template: /eng/pipelines/templates/jobs/archetype-sdk-client.yml
```
Using absolute path from the root make this stable.